kv/tikv/storage/kvstore/lock.go

```diff
@@ -54,3 +54,23 @@ func (lock *Lock) IsLockedFor(key []byte, txnStartTs uint64) bool {
 	}
 	return lock.TS <= txnStartTs
 }
+
+// LockedError occurs when a key or keys are locked. The protobuf representation of the locked keys is stored as Info.
+type LockedError struct {
```

Comment:
let's move all errors into one file?

Reply:
This error can't be with the others in commands because that would create an import cycle. I could pull it out into its own file in kvstore, but since there is just one error I thought it was better to put it here. I don't mind pulling it out though if you prefer.